### PR TITLE
Top-level `Rows()` and `Cols()` accessors.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: Sherlock/yoda/DOCU.md [hunk 1]
```diff
@@ -375,6 +375,70 @@ EXPECT_EQ("7,17,37,47,67,97",
             }
             return os.str().substr(1);
           }).Go());
+
+// Top-level per-row and per-columb accessors.
```
Comment (mzhurovich) [inline Sherlock/yoda/DOCU.md]: s/columb/column/

Comment (dkorolev) [inline Sherlock/yoda/DOCU.md]: 05d987a :+1:

File: Sherlock/yoda/DOCU.md [hunk 2]
```diff
@@ -375,6 +375,70 @@ EXPECT_EQ("7,17,37,47,67,97",
             }
             return os.str().substr(1);
           }).Go());
+
+// Top-level per-row and per-column accessors.
+api.Transaction([](PrimesAPI::T_DATA data) {
+  const auto accessor = MatrixEntry<PrimeCell>::Accessor(data);
+  EXPECT_EQ(10u, accessor.Rows().size());
+  EXPECT_EQ(6u, accessor.Cols().size());  // 2, 3, 5, 7 or 9, my captain.
```
Comment (mzhurovich) [inline Sherlock/yoda/DOCU.md]: Missing `1`.

Comment (dkorolev) [inline Sherlock/yoda/DOCU.md]: f218a14, my captain! :+1:

**Comment by mzhurovich** [comment]
@dkorolev :+1:

**Pull request merged**